### PR TITLE
[office-js, office-js-preview] Update Visio APIs to reflect current state of the CDN IntelliSense

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by Rick-Kirkham — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -117993,254 +117993,248 @@ declare namespace OneNote {
...
-         *
          * Visio Error code
          *
+         * @remarks
          * [Api set:  1.1]
          */
         errorCode: number;
         /**
-         *
          * Message about error that occured
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:118157]:
```suggestion
         * Message about error that occurred.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -118391,72 +118379,72 @@ declare namespace Visio {
...
          *
+         * @remarks
          * [Api set:  1.1]
          *
          * @eventproperty
          */
         readonly onDataRefreshComplete: OfficeExtension.EventHandlers<Visio.DataRefreshCompleteEventArgs>;
         /**
-         *
          * Occurs when there is an expected or unexpected error occured in the session.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:118391]:
```suggestion
         * Occurs when there is an expected or unexpected error occurred in the session.
```

File: types/office-js/index.d.ts [hunk 3]
```diff
@@ -109789,254 +109789,248 @@ declare namespace OneNote {
...
-     *
      * Provides information about the TaskPaneStateChanged event.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface TaskPaneStateChangedEventArgs {
         /**
-         *
          * Current state of the taskpane
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:109975]:
```suggestion
         * Current state of the task pane.
```

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -117993,254 +117993,248 @@ declare namespace OneNote {
...
-     *
      * Provides information about the TaskPaneStateChanged event.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface TaskPaneStateChangedEventArgs {
         /**
-         *
          * Current state of the taskpane
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:118179]: Should be "task pane" - here and other instances

```suggestion
         * Current state of the task pane.
```

File: types/office-js/index.d.ts [hunk 5]
```diff
@@ -109789,254 +109789,248 @@ declare namespace OneNote {
...
      * Represents the Application.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     class Application extends OfficeExtension.ClientObject {
         /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
         context: RequestContext;
         /**
+         * Show or hide the iframe application borders.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:109999]:
```suggestion
         * Shows or hides the iframe application borders.
```

File: types/office-js/index.d.ts [hunk 6]
```diff
@@ -109789,254 +109789,248 @@ declare namespace OneNote {
...
         /** Sets multiple properties on the object at the same time, based on an existing loaded object. */
         set(properties: Visio.Application): void;
         /**
-         *
          * Sets the visibility of a specific toolbar in the application.
          *
+         * @remarks
          * [Api set:  1.1]
          *
          * @param id The type of the Toolbar
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:110026]:
```suggestion
         * @param id The type of the Toolbar.
```

File: types/office-js/index.d.ts [hunk 7]
```diff
@@ -109789,254 +109789,248 @@ declare namespace OneNote {
...
          * @param show Whether the toolbar is visibile or not.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:110027]:
```suggestion
         * @param show Whether the toolbar is visible or not.
```

File: types/office-js-preview/index.d.ts [hunk 8]
```diff
@@ -117993,254 +117993,248 @@ declare namespace OneNote {
...
         set(properties: Visio.Application): void;
         /**
-         *
          * Sets the visibility of a specific toolbar in the application.
          *
+         * @remarks
          * [Api set:  1.1]
          *
          * @param id The type of the Toolbar
          * @param show Whether the toolbar is visibile or not.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:118231]:
```suggestion
         * @param id The type of the Toolbar.
         * @param show Whether the toolbar is visible or not.
```

File: types/office-js/index.d.ts [hunk 9]
```diff
@@ -110075,70 +110069,64 @@ declare namespace Visio {
...
          *
+         * @remarks
          * [Api set:  1.1]
          *
          * @param PageName Name of the page
          */
         setActivePage(PageName: string): void;
         /**
-         *
          * Show or Hide a TaskPane.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:110126]:
```suggestion
         * Shows or hides a TaskPane.
```

File: types/office-js/index.d.ts [hunk 10]
```diff
@@ -110187,72 +110175,72 @@ declare namespace Visio {
...
          *
+         * @remarks
          * [Api set:  1.1]
          *
          * @eventproperty
          */
         readonly onShapeMouseLeave: OfficeExtension.EventHandlers<Visio.ShapeMouseLeaveEventArgs>;
         /**
-         *
          * Occurs whenever a task pane state is changed
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:110241]:
```suggestion
         * Occurs whenever a task pane state is changed.
```

File: types/office-js-preview/index.d.ts [hunk 11]
```diff
@@ -118391,72 +118379,72 @@ declare namespace Visio {
...
          *
+         * @remarks
          * [Api set:  1.1]
          *
          * @eventproperty
          */
         readonly onShapeMouseLeave: OfficeExtension.EventHandlers<Visio.ShapeMouseLeaveEventArgs>;
         /**
-         *
          * Occurs whenever a task pane state is changed
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:118445]:
```suggestion
         * Occurs whenever a task pane state is changed.
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
I primarily gave my feedback in office-js-preview but the same should apply in office-js

File: types/office-js-preview/index.d.ts [hunk 12]
```diff
@@ -118279,93 +118273,87 @@ declare namespace Visio {
...
+                    This will be consumed by the DV Excel Add-In/Other third-party apps who embed the Visio drawing to show/hide the task pane.
          *
-         * Show or Hide a TaskPane.
-            This will be consumed by the DV Excel Add-In/Other third-party apps who embed the visio drawing to show/hide the task pane.
-         *
+         * @remarks
          * [Api set:  1.1]
          *
          * @param taskPaneType Type of the 1st Party TaskPane. It can take values from enum TaskPaneType
          * @param initialProps Optional Parameter. This is a generic data structure which would be filled with initial data required to initialize the content of the Taskpane
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:118337]:
```suggestion
         * @param initialProps Optional Parameter. This is a generic data structure which would be filled with initial data required to initialize the content of the task pane.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**